### PR TITLE
day-4-typo-check

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 4/04_00_ARC04_4_1B.txt [hunk 1]
```diff
@@ -46,7 +46,7 @@ Over there―――
 [sha:099f11d74de9e12d6be2210295ef62c594d035ad]{
 -- Page 260, Offset 6284.
 -- 吸血鬼を名乗る女以上に、人間の都市に居てはならない生き物がいた。
-Is something even more outlandish than the woman who calls herself a vampire.
+Is something even more outlandish than the woman who calls herself a vampire?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_4_1B.txt:49]: Not a question?

File: script/Arcueid/Day 4/04_00_ARC04_6_2B.txt [hunk 2]
```diff
@@ -377,7 +377,7 @@ She claps her hands together, smiling at me.
 [sha:9cd9f4ccc4361e71de3fbdc119ae578590745992]{
 -- Page 498, Offset 6932.
 -- 通りすがりの殺人鬼への当てつけであろう。たいへん居心地が悪い。
-It seems that she's taking a dig at said murderer. It doesn't feel great.
+It seems that she's taking a dig at the said murderer. It doesn't feel great.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_6_2B.txt:380]: Revert

File: script/Arcueid/Day 4/04_00_ARC04_6_2D.txt [hunk 3]
```diff
@@ -412,7 +412,7 @@ I'm not kidding when I say that your eyes really are that bad!"
 [sha:aa086cf3af750f0a69b4cacb438d2d5cc91323c3]{
 -- Page 594, Offset 7161.
 -- どんなに長い寿命、どんなに強大な生命にだってこれはある。たとえ宇宙の寿命より長い寿命を持つ生命があるとしても、それは無限に生き続ける訳じゃない」
-Nobody can escape this fate, no matter how long-lived they are or how mighty their existence.
+Nobody can escape this fate, no matter how long-lived they are or how mighty their existence is.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_6_2D.txt:415]: Revert

File: script/Arcueid/Day 4/04_00_ARC04_9_1.txt [hunk 4]
```diff
@@ -53,7 +53,7 @@ The only thing still lit is the bank of elevators at the end of the hall.
 -- Page 725, Offset 7511.
 -- 部屋にいた時より喉が痛む。俺を包む空気はやすりのようにざらついて、熱かった。
 My throat hurts even more than it did when I was in the room.
- The air surrounding me is hot, and rough as sandpaper.
+ The air surrounding me is hot and rough as sandpaper.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_9_1.txt:56]: Keep comma, sandpaper isn't hot

File: script/Arcueid/Day 4/04_00_ARC04_10.txt [hunk 5]
```diff
@@ -947,7 +947,7 @@ Did Arcueid's words strike some nerve in the man?
 [sha:06f3136615bbfd96ae67e5089227ad6cc31b42f0]{
 -- Page 866, Offset 7962.
 -- 炎をまき散らす吸血鬼は悩ましげに、
-A troubled expression on his face, the flame-spreading vampire speaks―――
+With a troubled expression on his face, the flame-spreading vampire speaks―――
```
Comment (rschlaikjer) [inline script/Arcueid/Day 4/04_00_ARC04_10.txt:950]: Maybe prefer orig, but it's basically style preference

**Review by rschlaikjer — APPROVED** [review]

File: script/Arcueid/Day 4/04_00_ARC04_6_2D.txt (hunk 3 above)
Comment (yk1512) [reply script/Arcueid/Day 4/04_00_ARC04_6_2D.txt:415]: I think the 'is' makes sense to put in.

**Review by yk1512 — APPROVED** [review]

File: script/Arcueid/Day 4/04_00_ARC04_6_2D.txt (hunk 3 above)
Comment (Avuxo) [reply script/Arcueid/Day 4/04_00_ARC04_6_2D.txt:415]: I'm with Ross. Reads nicer without

File: script/Arcueid/Day 4/04_00_ARC04_9_1.txt [hunk 6]
```diff
@@ -390,7 +390,7 @@ At the end of the hallway, inside the elevator, the dogs are scavenging off peop
 -- Page 741, Offset 7577.
 -- エレベーターの隙間、下の階に通じる通風口から。
 // Rearranged slightly across these 2 lines
-Inside of the elevator is a vent leading to the floor below.
+Inside the elevator is a vent leading to the floor below.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_9_1.txt:393]: I prefer the "of" even if it forces a pause.

**Review by Avuxo — APPROVED** [review]

**Pull request merged**